instagram-app/src/App.js

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-import SearchBar from "./components/SearchBar/SearchBar.js"
-import PostContainer from "./components/PostContainer/PostContainer.js"
-
-import DummyData from "./dummy-data"
+import React from "react";
+import "./App.css";
+import SearchBar from "./components/SearchBar/SearchBar.js";
+import PostContainer from "./components/PostContainer/PostContainer.js";
+import CommentSection from "./components/CommentSection/CommentSection.js";
+import dummyData from "./dummy-data.js";
 
 function App() {
   return (
     <div className="App">
-     <h1>Hello World</h1>
-     <SearchBar/>
-     <PostContainer/>
+      <h1>Hello World</h1>
+      <SearchBar />
+      
+        {dummyData.map( element => {
+          return (
+          <div>
```

Comment:
you should add a  unique key={} every time you map over data in JSX, for you to get rid of that warnings in the console